### PR TITLE
PROD-800 #2512  - Fix Document description doesn't carry over from group feed

### LINKED ISSUE #2512: Document description doesn't carry over from group feed
**Describe the bug**
When adding a Document to a Group Feed, we noticed that the Document goes automatically to the Documents tab (which is great!) but the description doesn't get carried over with it.

**To Reproduce**
http://somup.com/cr1b18q5KF

**Expected behavior**
The description should be carried over from feed to document preview window.

**Support ticket links**
https://secure.helpscout.net/conversation/1539799780/147168?folderId=3701248

### PR DESCRIPTION
**Pull request by milanhirapra**
* [x] Have you followed the [Pull Requests Guidelines](https://github.com/buddyboss/buddyboss-platform/wiki/Submitting-Pull-Requests#pull-request-guidelines)?
* [x] Does your code follow the [WordPress' Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Fixes #2512 .

### How to test the changes in this Pull Request:

1. Go to the Activity page
2. Create a new activity with Media/Document/Video and content
3. Click on Media/Document/Video and can see the activity content

### Proof Screenshots or Video
https://vimeo.com/596598709/9cb3dc56fd

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### CODE REVIEW

File: src/bp-activity/bp-activity-filters.php [hunk 1]
```diff
@@ -2927,3 +2930,40 @@ function bb_activity_delete_link_review_attachment( $activities ) {
 		}
 	}
 }
+
+/**
+ * Add activity description to activity media/document/video
+ *
+ * @param $media
+ */
+function bb_activity_update_description( $media ) {
```
Comment (dipskakadiya) [inline src/bp-activity/bp-activity-filters.php:2939]: @milanhirapra Function name should be `bb_activity_update_media_video_document_description`

**Review by dipskakadiya — CHANGES_REQUESTED** [review]
@milanhirapra Minor change requested.

File: src/bp-activity/bp-activity-filters.php (hunk 1 above)
Comment (milanhirapra) [reply src/bp-activity/bp-activity-filters.php]: @dipskakadiya I made this change

**Review by dipskakadiya — APPROVED** [review]

**Review by dipskakadiya — DISMISSED** [review]

File: src/bp-activity/bp-activity-filters.php [hunk 2]
```diff
@@ -516,14 +519,14 @@ function bp_activity_make_nofollow_filter( $text ) {
...
-	 *
-	 * @since BuddyPress 1.2.0
-	 *
-	 * @param array $matches Items matched by preg_replace_callback() in bp_activity_make_nofollow_filter().
-	 * @return string $text Link with rel=nofollow added.
-	 */
+/**
+ * Add rel=nofollow to a link.
+ *
+ * @since BuddyPress 1.2.0
```
Comment (chetansatasiya) [inline src/bp-activity/bp-activity-filters.php:525]:
```suggestion
 * @since BuddyBoss [BBVERSION]
```

**Review by chetansatasiya — CHANGES_REQUESTED** [review]
Need changes

**Pull request closed without merging**